src/rf/apps/workers/process.py

```diff
@@ -58,48 +63,84 @@ def start_polling(self):
                         delete_message = self.check_timeout(record['data'])
                     elif job_type in JOB_VALIDATE:
                         if record['eventSource'] == 'aws:s3':
-                            delete_message = self.validate_image(record)
+                            delete_message = self.__validate_image(record)
```

Comment:
We may want to add better error handling on the dispatcher, so if any of these job types throw an unexpected error, it doesn't crash the agent.